SignalShareExtension/ShareViewController.swift

```diff
@@ -15,21 +15,14 @@ public class ShareViewController: UINavigationController, ShareViewDelegate, SAE
     private var hasInitialRootViewController = false
     private var isReadyForAppExtensions = false
 
-    var loadViewController: SAELoadViewController!
+    var progressPoller: ProgressPoller?
+    var loadViewController: SAELoadViewController?
 
     override open func loadView() {
         super.loadView()
 
         Logger.debug("\(self.logTag) \(#function)")
 
-        // We can't show the conversation picker until the DB is set up.
-        // Normally this will only take a moment, so rather than flickering and then hiding the loading screen
-        // We start as invisible, and only fade it in if it's going to take a while
-        self.view.alpha = 0
```

Comment:
I simplified the "Loading UI" logic. The Loading VC is now presented modally (after a short delay) and all this animation stuff went away.